lib/trader.js

```diff
@@ -480,6 +480,24 @@ Trader.prototype.pollError = function pollError (err) {
   this.emit('networkDown')
 }
 
+Trader.prototype.networkHeartbeat = function networkHeartbeat (obj) {
+  return this.request({
+    path: '/network/heartbeat',
+    body: obj,
+    method: 'POST'
+  })
+    .catch(err => console.log('Failed to send network heartbeat', err))
```

Comment:
Maybe it's better to use `console.error`, what do you think?

Reply:
Correct